knowledge-portal/central/initiate-push-to-source.sh

```diff
@@ -6,15 +6,21 @@ set -e
 git config --global user.email "PlatformTeam@detecttechnologies.com"
 git config --global user.name "Detect Gitlab Bot"
 
-# changed files can be obtained by git diff between latest commit and unstaged area
-git reset --soft HEAD~1
-changed_files=$(git diff --name-only HEAD)
+# Install git lfs
+apt-get update -qq && apt-get install -y -qq git-lfs
+git lfs install
+git lfs track "*.jpg" "*.jpeg" "*.png" "*.gif" "*.svg" "*.ico" "*.mp3" "*.mp4"
```

Comment:
Is it really needed to run git lfs install and track on every commit? Tracking would persist in the .gitattribute file.